README.md

```diff
@@ -256,6 +256,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [First Citdael](http://www.skyislandsvr.com/pages/firstcitadel.html) - Citadel with flowing waterfalls
 - [Missed Connections](http://armthethinker.github.io/webVR-experiments/5--missed-connections.html) - Visualization of Craigslist "Missed Connections" section
 - [Aeon](http://aeon.horoman.com/) (by [@h0r0man]) - Aeon is a 3D environment with VR support, where the user runs an island that tells about the prequel to the main story of Aeon.
+- [SlideAmaze](http://slideamaze.com/) - Turns your slideshow presentation [into a 3D maze](http://fouryears.eu/2016/07/27/slide-a-maze/).
```

Comment:
There has been a lot of discussion about requiring secure origins (i.e., HTTPS) for WebVR [in the WebVR Mailing List](https://mail.mozilla.org/pipermail/web-vr-discuss/2016-July/thread.html) (`http://localhost:*` and `http://127.1.*.*` _are_ considered secure, FYI). And official announcements from the Chrome Security and Firefox Security teams are coming soon to the WebVR mailing list, Twitter accounts, WebVR sub-Reddit, IsWebVRReady.org, etc.

Here are some helpful links:

* https://www.chromium.org/Home/chromium-security/prefer-secure-origins-for-powerful-new-features
* https://blog.mozilla.org/security/2015/04/30/deprecating-non-secure-http/
* https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts
* https://w3c.github.io/webappsec/specs/powerfulfeatures/
* https://sites.google.com/a/chromium.org/dev/Home/chromium-security/deprecating-powerful-features-on-insecure-origins

FWIW, Chrome already has removed support for HTTP-only sites for getUserMedia, Geolocation, and Service Workers (which always had the HTTPS restriction from initial spec + implementations).

These are indeed totally both nitpicks I brought up, and I didn't mean to reject anything by the suggestions I made. And, I'm sure you know, but today if a site is `https` and it wants to embed your `http` site using an `<iframe>` it cannot because of the browser's Mixed Content Policy.